### PR TITLE
Code correct

### PR DESCRIPTION
**Pull request by manavm1990**
IDK if this is intentional, but the `radio` buttons _all_ have `checked` attribute, which, of course isn't usually the case for `radio`.

### CODE REVIEW

**Comment by manavm1990** [comment]
[MDN example.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio)

**Comment by manavm1990** [comment]
`<==` and `>==` should note have `==`

File: docs/Wk1Mod4.md [hunk 1]
```diff
@@ -205,7 +243,7 @@ The layout above is a very common layout for blogs... much like the blog that we
 2. Let's change the unordered list (`<ul>`) of navigation links into an actual horizontal bar instead of a vertical bullet-pointed list. Try the following in your navigation bar (with a `class` of `navigation`, for this example):
 
 ```css
-#navigation > ul > li {
+nav > ul > li {
```
Comment (NAlexPear) [inline docs/Wk1Mod4.md]: we've been using `div`s and `id`s for consistency, but you're right that `nav` is probably better long-term.

File: docs/Wk2Mod1.md [hunk 2]
```diff
@@ -49,37 +49,41 @@ What's the answer? It's definitely NOT 4 like you or I might expect. And that's
...
+    + Our **keyword** and the name of the variable are separated by a space, and the variable name can't include spaces
     + variable values can be any data type (that we've covered so far)
     + variable values are assigned using the *assignment operator* (which looks like an equals sign to you and I).
     + the assignment 'phrase' ends with a semicolon. We end JavaScript 'sentences' with semicolons... always!
     + the REPL outputs *undefined* after we define the variable. That's expected, since we're only saving that variable to memory.
     + We can access the value stored to the variable name by typing in that variable's name to the REPL.
     + To actually log the value to the Dev Console (much more useful in real-world applications), we can use the built in function `console.log([variable name])`
     + When we use the variable, we DON'T use quotation marks. Quotes mean Strings, and variables are variables, not strings of text.
+    + `const` indicates a value that we can't replace in memory. It's basically a 'constant.' So, if we do `const currentYear = 2017`, we will get an error.
```
Comment (NAlexPear) [inline docs/Wk2Mod1.md]: I'd hesitate to call `const` a constant. Consider:

```
const dog = { name: "fido" };

dog.name = "rufus";

console.log(dog); // { name: "rufus" }
```
Perhaps `const means that we can't replace the entire value in memory, although we'll see that we can change parts of complex data later on`?

File: docs/Wk2Mod1.md [hunk 3]
```diff
@@ -49,37 +49,41 @@ What's the answer? It's definitely NOT 4 like you or I might expect. And that's
...
+    + `let` and `var` __do__ allow the value to be replaced.
+    + `var` initializes all variables as `undefined` prior to JS reviewing and assigning the value we actually want.
```
Comment (NAlexPear) [inline docs/Wk2Mod1.md]: Isn't this true of all `let`, too? i.e.:

```
let foo; // undefined
```

File: docs/Wk3Mod2.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 The `while` loop looks a lot like an `if` statement. They both execute their associated code block based on the result of their conditional expression. The difference being, the `while` loop will repeatedly check its conditional expression and continue to run its code block as long as it evaluates to `true`. Give it a try:
 
 ```javascript
-var n = 0;
+const n = 0;
```
Comment (NAlexPear) [inline docs/Wk3Mod2.md]: A lot of these `const`s should be `let`s!

File: projects/Rock-Paper-Scissors.md [hunk 5]
```diff
@@ -28,13 +28,13 @@ Use `prompt()` to create a Rock, Paper, Scissors game for visitors to your portf
...
-var userChoice = prompt("Choose Rock, Paper, or Scissors by typing 'R', 'P', or 'S'");
+const userChoice = prompt("Choose Rock, Paper, or Scissors by typing 'R', 'P', or 'S'");
 ```
 2. Use `Math.random()` to choose a play for the computer after you've gathered input. HINT:
 
 ```javascript
-var rng = Math.random();
-var computerChoice = "R";
+const rng = Math.random();
+const computerChoice = "R";
```
Comment (NAlexPear) [inline projects/Rock-Paper-Scissors.md]: This should be a `let` (assuming we even want to keep this activity in here... it's an oldie, for sure)

File: docs/Wk1Mod4.md (hunk 1 above)
Comment (manavm1990) [reply docs/Wk1Mod4.md]: Yeah, we using `<nav>`, `<main>` and `<footer>` and `<header>`.

File: docs/Wk2Mod1.md (hunk 3 above)
Comment (manavm1990) [reply docs/Wk2Mod1.md]: I think I was trying to imply about 'hoisting'...but maybe that's not appropriate here.

File: docs/Wk3Mod2.md (hunk 4 above)
Comment (manavm1990) [reply docs/Wk3Mod2.md]: 🆗 Working on these. I think I am getting them all, but some of these will not be apparent just from reading but will become apparent when we go to run them.

File: projects/Rock-Paper-Scissors.md (hunk 5 above)
Comment (manavm1990) [reply projects/Rock-Paper-Scissors.md]: Not 'line 36', is it?

**Comment by NAlexPear** [comment]
Thanks for making these updates! Much appreciated :pray:

**Pull request merged**